.github/workflows/UnitTests.yml

```diff
@@ -1,11 +1,20 @@
-name: Unit Tests
+name: MPI.jl Unit Tests
 
 on:
   pull_request:
-
+    paths:
```

Comment:
We should probably also include this file in the list, so that when this is modified these tests are also run?